### PR TITLE
adapt to new rubocop

### PR DESCRIPTION
**Pull request by jreidinger**
disable one of new style checks for Struct.new

### CODE REVIEW

File: test/addon_spec.rb [hunk 1]
```diff
@@ -171,8 +170,8 @@
   describe "#selectable?" do
     let(:addons) do
       Registration::Addon.find_all(double(
-          get_addon_list:     [addon_with_child_generator],
-          activated_products: []
+                                     get_addon_list:     [addon_with_child_generator],
+                                     activated_products: []
```
Comment (mchf) [inline test/addon_spec.rb]: I'd expect:
```
find_all(
  double(
    get_...
    activated_...
  )
)
```
and definitely not so big indentation of params

File: test/helpers_spec.rb [hunk 2]
```diff
@@ -191,10 +191,10 @@
...
 
       addon = Registration::Addon.new(addon_generator(
-          "zypper_name"  => "sle-sdk",
-          "version"      => "12",
-          "arch"         => "x86_64",
-          "release_type" => nil
+                                        "zypper_name"  => "sle-sdk",
+                                        "version"      => "12",
+                                        "arch"         => "x86_64",
+                                        "release_type" => nil
```
Comment (mchf) [inline test/helpers_spec.rb]: same as above

File: test/addon_spec.rb (hunk 1 above)
Comment (jreidinger) [inline test/addon_spec.rb]: yep, it is indended to align method call, so your one with double is nicer. This is just auto-correct.

**Comment by jreidinger** [comment]
@mchf updated

**Comment by mchf** [comment]
LGTM

**Comment by jreidinger** [comment]
thanks

**Pull request merged**